distutils.rst

```diff
@@ -296,9 +296,11 @@ setuptools_scm (flit_scm, hatch-vcs, pdm-backend) and snapshots
 setuptools_scm_ is a package providing additional features for running
 inside a VCS checkout, in particular the ability to determine version
 from VCS tags.  However, this works correctly only when the package
-is built from VCS checkout or an ``sdist`` archive containing
-pregenerated metadata.  It does not work when building from a GitHub
-snapshot::
+is built from a VCS checkout, or an ``sdist`` archive containing
+pregenerated metadata, or the repository is configured to support
+setuptools_scm installing from git archives in which case
+a ``.git_archival.txt`` file will be present.
+When building from a GitHub snapshot does not work::
```

Comment:
This has gotten too long to be easily understood. How about something like: "However, this may not work when building from a GitHub snapshot::"